pre-registration/pre-registration-datasync-service/src/main/java/io/mosip/preregistration/datasync/service/util/DataSyncServiceUtil.java

```diff
@@ -259,8 +253,8 @@ public boolean validateReverseDataSyncRequest(ReverseDataSyncRequestDTO reverseD
 	 * @param regCenterId
 	 * @return preRegIdsByRegCenterIdResponseDTO
 	 */
-	public BookingDataByRegIdDto getBookedPreIdsByDateAndRegCenterIdRestService(String fromDate,
-			String toDate, String regCenterId) {
+	public BookingDataByRegIdDto getBookedPreIdsByDateAndRegCenterIdRestService(String fromDate, String toDate,
+			String regCenterId) {
 		log.info("sessionId", "idType", "id", "In callGetPreIdsRestService method of datasync service util");
```

Comment:
Why do we need this? Can we rather print the prereg id?

Reply:
corrected the logs